libs/core/src/lib/grid-list/components/grid-list-item/grid-list-item.component.ts

```diff
@@ -118,6 +118,12 @@ export class GridListItemComponent<T> implements OnChanges, AfterViewInit, OnDes
     @Input()
     isNavigated = false;
 
+    /**
+     * The active state of the list item.
+     * If set to true, the whole card has active state. Became false only on click edit button.
```

Comment:
If set to true, the whole card has active state. Becomes false only when the Edit button is clicked.

Reply:
we should add `@hidden` as well